src/layouts/HeaderLayout.tsx

```diff
@@ -17,14 +17,6 @@ const variants = {
 const HeaderLayout: React.FC<HeaderLayoutProps> = ({ children, title }) => {
   const headName = `${title} | CourtCanva`;
   return (
-    <motion.article
-      initial="hidden"
-      animate="enter"
-      exit="exit"
-      variants={variants}
```

Comment:
<img width="445" alt="image" src="https://user-images.githubusercontent.com/61776449/174044963-22a58c0b-ef72-4b2c-a259-797141f1396b.png">
Looks like we don't need variants now